pages/en/network/developing.mdx

```diff
@@ -2,36 +2,39 @@
+Developers can build subgraphs, open APIs, and deploy them locally or in the [Subgraph Studio](https://thegraph.com/studio/).
 
-The Graph CLI tool streamlines subgraph creation. When you initialize your subgraph, all the default files will be created & configured. Developers can use the built-in APIs or add other libraries as dependencies as long as they are compatible with [AssemblyScript](https://github.com/AssemblyScript/assemblyscript). Subgraphs should be thoroughly tested before deployment. Developers can use [MatchStick](https://github.com/LimeChain/matchstick) to test the mapping logic in a sandboxed environment. Once your subgraph is built & tested, it can be deployed to The Graph's decentralized network. However, before anyone can begin querying it, your subgraph must be curated & indexed.
+Once published and indexed to the The Graph Network, anyone can query the subgraph with [GraphQL](https://graphql.org/learn/schema-language/) and use it in their protocol or applications.
 
 ### Curation
 
-Curators use their knowledge of web3 to assess and signal new subgraphs. They look for subgraphs with reliable and relevant data.
+Subgraph developers often curate their own subgraph. This proves to Indexers that they've published a high-quality subgraph to the network and attracts additional Curators.
 
 Curators stake GRT with the subgraph and receive curation shares, which guarantee a percentage of future query fees. The curation signal increases as more Curators signal on the subgraph.
 
-A strong curation signal is crucial to a subgraph. Indexers use the curation signal to determine which subgraphs to index to the network, making it available for querying.
+A strong curation signal is crucial because Indexers use the curation signal to determine which subgraphs to index to the network, making it available for querying.
 
-### Deploying on the network
+### Publishing on the Network
 
-By deploying your subgraph to The Graph network, you're making that data available to developers and users across the web3 ecosystem. Anyone will be able to query it.
+By deploying your subgraph to The Graph Network, you're making that data available to developers and users across the web3 ecosystem. Anyone will be able to query it.
 
 Data from your subgraph can be used to improve user experience, build new features, or even create entirely new applications.
 
 ## Querying Subgraphs
 
-Developers that query subgraphs serve it to users through a simple interface. They can use the GraphQL API to query data directly from the blockchain. Developers can manipulate, sort, or aggregate the data any way they want.
+Developers that query subgraphs serve it to users through a simple interface. They can use the GraphQL API to query data from the subgraph's processed database. Developers can manipulate, sort, or aggregate the data any way they want.
```

Comment:
I don't know if the `through a simple interface` is clear.

Reply:
Good catch - reworded this for clarity